compiler/src/dmd/func.d

```diff
@@ -1572,6 +1578,12 @@ extern (C++) class FuncDeclaration : Declaration
         bool gag = false, Loc loc = Loc.init, const(char)* fmt = null,
         RootObject arg0 = null, RootObject arg1 = null, RootObject arg2 = null)
     {
+        if (safetyInprocess && semanticRun < PASS.semantic3 && _scope)
+        {
+            this.semantic2(_scope);
+            this.semantic3(_scope);
```

Comment:
Coverage

Reply:
I added that up, but `nothrow` doesn't seem to take effect. Let me test it locally.